docs/customize/vocabularies/users.md

```diff
@@ -73,8 +73,15 @@ db.session.commit()
 
 ## Confirm user
 
-Only confirmed accounts can be logged in. To confirm an account programmatically, create a new shell using `pipenv run invenio shell` and
-run:
+Only confirmed accounts can be logged in. There are two ways to confirm an account as an administrator:
+
+When creating the account pass the `--confirm` parameter like so:
+```shell
```

Comment:
Let's pick `bash` or `shell` for all appropriate. I want to say `shell`: it gives just as good contrast and might save us from "bashisms", but I don't know / not a big deal as long as it's just the same one.